src/explorer/App.js

```diff
@@ -112,6 +112,9 @@ export function createApp(
             {spacer()}
             <Link href={feedbackUrl}>feedback</Link>
           </p>
+          <ProjectDetail
```

Comment:
If we're going to have a ProjectDetail component, then that component should have the logic for how we construct a title for a given repoId. Otherwise, we could just directly create the <p> here.

So, I suggest one of the following two changes:

- Create a ProjectDetail component which takes a `RepoId` as a prop, and then uses that to render a title
OR
- Just directly create a <p> here with the right title, without the indirection of an additional component.

I have a preference for option 1 as it will scale better in complexity as we add more "project detail" type information (which I imagine we will)

Reply:
Yup!